packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -52,6 +50,8 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
   const [visitTime, setVisitTime] = useState(dayjs(new Date()).format('hh:mm'));
   const [visitType, setVisitType] = useState<string | null>(null);
   const state = useMemo(() => ({ patientUuid }), [patientUuid]);
+  // @ts-ignore
+  const { updateCurrentVisit } = useVisit();
```

Comment:
How is `useVisit` supposed to work with no `patientUuid` argument? The function is typed to require `patientUuid`, and seems written to require it. Perhaps `updateCurrentVisit` should just be independent of `useVisit`?